Deploy/hooks/validate.sh

```diff
@@ -2,10 +2,10 @@
 
 case $DEPLOYMENT_GROUP_NAME in
   "production")
-    PORT=80
+    PORT=8080
     ;;
   "staging")
-    PORT=8080
+    PORT=8081
```

Comment:
we need to note how these are set up in documentation because things could break if we ever move from ec2

Reply:
Not sure how to note this. 🤔 This is just a kind of configuration thing. We just say here 
> If this is a staging build, create a binding of port `80` to port `8081` on the local machine.
> If this is a production build, bind to `8080`

The same ports are used in Nginx configuration, `api.oneleif.com` will proxy requests to `127.0,0.1:8080` and `dev.api.oneleif.com` will proxy to `127.0.0.1:8081`.

This is not related to the platform/provider we are using. This can work in the same way on your local machine.